scipy/spatial/src/distance_wrap.c

```diff
@@ -32,6 +32,16 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+/* avoid unknown pragma warning in clang which may be used by developers */
+#ifndef __clang__
```

Comment:
Shouldn't this be ``#ifdef __GNUC__ >= 4 && (__GNUC_MINOR__ > 4``? Avoid warnings for other compilers and older gcc versions?